src/Orleans.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25420.1
```

Comment:
this maybe inconvenient to you. I wonder if there's a way to turn it off. 

But avoiding unnecessary change is first thing I learnt as a SDE. And this is a common standard based on my pass experience with couple teams in Amazon. it decrease work for your reviewer and people in the future who will be looking at this commit. 

Don't take it personal. I'm just apply the same standard to you as to other people and as other people to me. And just for the record, I asked a core team member, he does it too.